### DISCUSSION TITLE
Is Haali Splitter still worth listing in Apps Manager?

### DISCUSSION THREAD

**Discussion by JJKylee** [staxrip/staxrip#466] [General]
I'm wondering how many people are still using it along with _eac3to_.
These days MKV related jobs are superbly done by MKVToolNix tools like `mkvextract`, `mkvmerge`, and so on.

It's been more than 7 years since it was updated and it's known to have some problems in recent days.
Furthermore, it doesn't seem to be developed any more.

I see no reason why StaxRip should go along with this paleolithic tool. 😅
But please correct me if I'm wrong.

**Comment by ghost** [comment]
> Is Haali Splitter still worth listing in Apps Manager?

Obviouly, it's not 🤣  I can't find a single reason to keep it! I challenge anyone who can 😆

**Comment by Dendraspis** [comment]
Haali is still used for demuxing m2ts files with eac3to, where the user can decide which output format the video stream should have.

Removing this option would make Haali unnecessary, but for this case Haali should work without issues. 🤔

**Reply by JJKylee** [reply]
Yeah, I know.
But in that regard, can't `DGIndex` replace `eac3to`, apart from its performance efficiency? Considering that `eac3to` is also very much under-maintained (or seemingly abandoned by its author madshi), I'm just wondering how effective they are in reality and how many people are actively using both. 🤔

**Reply by Dendraspis** [reply]
I use *eac3to* multiple times a day. 😲 

*eac3to* is used to demux M2TS files and automatically convert audio streams (if wished). Switching to *DGIndex* would mean a lot of changes code-wise and it would be somehow also pointless, because the stream can be extracted anyhow.

Nevertheless *Haali* is marked as *not required*, so why not keeping it if anybody wants to use it?!

**Reply by JJKylee** [reply]
I see. You’re right. Leaving it as it is does not harm, but I was simply curios cuz I stopped using `eac3to` since like couple years ago, let alone Haali, and was doubtful if those two tools would ever be updated again. 😝

**Reply by ghost** [reply]
Well, then as you say, it won't harm to leave it. I knew eac3to is still useful, but I had no idea Haali could still be used.
Thanks @Dendraspis for your enlightenment 💡

**Reply by Dendraspis** [reply]
> [..] cuz I stopped using `eac3to` since like couple years ago, let alone Haali, and was doubtful if those two tools would ever be updated again. 😝

The good old times ... 🤗

> I knew eac3to is still useful, but I had no idea Haali could still be used.

The good old times ... 😁😆
